dts/bindings/misc/nxp,s32-emios.yaml

```diff
@@ -0,0 +1,95 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  NXP S32 Enhanced Modular IO SubSystem (eMIOS) node for S32 SoCs.
+  eMIOS provides independent unified channels (UCs), some of channels
+  have internal counter that either can be used independently or used
+  as a reference timebase (master bus) for other channels.
+
+compatible: "nxp,s32-emios"
+
+include: [base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  clock-divider:
+    type: int
+    required: true
+    description: |
+      Clock divider value for the global prescaler. Could be in range [1 .. 256]
+
+child-binding:
+  child-binding:
+    description: |
+      Node for eMIOS master bus. Each channel is capable to become a master bus has
+      a node defined in root devicetree but is disabled by default. To allow using
+      the master bus, the devicetree node should be enabled and dts properties
+      should be configured as required by application. For example, to enable bus A
+      of eMIOS instance 0:
+        master_bus {
+          emios0_bus_a: emios0_bus_a {
```

Comment:
There are more properties required than the ones included in this snippet. Perhaps add them all to have a complete example.

Reply:
Yep, i added for more details